src/main/java/org/jabref/model/entry/BibtexSingleField.java

```diff
@@ -8,7 +8,8 @@
  */
 public class BibtexSingleField {
 
-    public static final int DEFAULT_FIELD_LENGTH = 100;
+    // TODO: This constant should be moved to the gui package, probably to MainTableColumnFactory
+    public static final double DEFAULT_FIELD_LENGTH = 100;
```

Comment:
Would it be possible to move this as suggested in the comment?

Reply:
This is not possible at the moment since the field is still used in other places in model. The new main table is free from this dependency but the import and search dialog still rely on it as the have before.

Reply:
Ok, no problem. Then just leave as it is.